src/index.js

```diff
@@ -0,0 +1,7 @@
+import getAuthenticatedAPIClient from './AuthenticatedAPIClient';
+import PrivateRoute from './PrivateRoute';
+
+export {
+  getAuthenticatedAPIClient,
+  PrivateRoute,
+};
```

Comment:
Question: do micro-frontends need to call any of these libraries at all?  Or does Axios allow you to integrate into the system without any explicit calls by the micro-frontend?

Reply:
Yes, frontend apps will need to reference this code. See https://github.com/edx/edx-portal/pull/62 for example usage. Also, I'm still planning to add to the README explaining how to use this package.